src/Command/UnmakeEntityCommand.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Hexis\UnmakeBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+class UnmakeEntityCommand extends Command
+{
+    protected static $defaultName = 'unmake:entity';
+    protected static $defaultDescription = 'Delete entities based on provided filters.';
+    private EntityManagerInterface $entityManager;
+    private ParameterBagInterface $parameterBag;
+
+    public function __construct(EntityManagerInterface $entityManager, ParameterBagInterface $parameterBag)
+    {
+        parent::__construct();
+        $this->entityManager = $entityManager;
+        $this->parameterBag = $parameterBag;
+    }
+
+    protected function configure(): void
+    {
+        $this->setDescription(self::$defaultDescription);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $entityList = $this->getEntityNames();
+        $entitySelected = $this->selectEntity($input, $output, $entityList);
+
+        if (!$entitySelected) {
+            $output->writeln('<error>No valid entity selected.</error>');
+            return Command::FAILURE;
+        }
+
+        $qb = $this->entityManager->createQueryBuilder();
+        $qb->select('a')->from($entitySelected, 'a');
+
+        $filterCounter = 0;
+        do {
+            $filterCounter = $this->addFilter($input, $output, $qb, $filterCounter);
+        } while ($this->askForAnotherProperty($input, $output));
+
+
+        $result = $qb->getQuery()->getResult();
+        $output->writeln('<info>Number of rows affected:</info> ' . count($result));
```

Comment:
There is a helper `->info` and I would usually make use of `sprintf` to format the message without concatenation.